docs/pressable.md

```diff
@@ -0,0 +1,184 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a Core Component wrapper that can detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```
+import React, { useState } from 'react';
+import {
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+export default function App() => {

```

Comment:
`export default App = () => {` is the correct syntax. Current code produces `Unexpected token` error.

Reply:
thank you! I fixed it.